scripts/zpool

```diff
@@ -53,7 +53,22 @@ zpool_monitor() {
     # exit immediately if configuration is not valid
     zpool_validate_all || exit $?
 
-    ocf_run -q -info zpool status $OCF_RESKEY_pool
+    if [ -z "$OCF_RESKEY_pool" ] ; then
+        # We have no pool name
+        false
+    elif [ -e /proc/spl/kstat/zfs/$OCF_RESKEY_pool/state ] ; then
+        # Best case, we can read the pool health locklessly from the proc file.
+        state=$(cat /proc/spl/kstat/zfs/$OCF_RESKEY_pool/state)
+        [ "$state" != "SUSPENDED" ] && [ "$state" != "FAULTED" ] && [ "$state" != "UNAVAIL" ]
```

Comment:
Do _all_ of these states really mean that we should power off the node and try starting the zpool on another node?  Maybe they do, I don't know.  But we should be very careful that this is what we want.

Reply:
Also, I would like to see each of these conditionals explicitly set $rc, and log a _different_ message with ocf_log.  That will allow users to tell which of the methods was used to determine the zpool status.